frontend/app/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "streamlit-browser",
+  "name": "@streamlit/app",
```

Comment:
I guess this could be a separate PR but simply included because it's a tiny change.